SignalServiceKit/protobuf/OWSSignalServiceProtos.proto

```diff
@@ -158,14 +159,19 @@ message SyncMessage {
     optional uint64 timestamp = 2;
   }
 
-  optional Sent     sent     = 1;
-  optional Contacts contacts = 2;
-  optional Groups   groups   = 3;
-  optional Request  request  = 4;
-  repeated Read     read     = 5;
-  optional Blocked  blocked  = 6;
-  optional Verified verified = 7;
-  optional bytes    padding  = 8;
+  message Configuration {
+    optional bool readReceipts = 1;
+  }
+
+  optional Sent          sent          = 1;
+  optional Contacts      contacts      = 2;
+  optional Groups        groups        = 3;
+  optional Request       request       = 4;
+  repeated Read          read          = 5;
+  optional Blocked       blocked       = 6;
+  optional Verified      verified      = 7;
+  optional Configuration configuration = 9;
```

Comment:
why are the fields out of order?

Reply:
It's really easy to introduce schema issues through reusing the index of a deprecated field.

Reply:
Yeah, see https://github.com/WhisperSystems/libsignal-service-java/commit/7123cf57fe80f8199802f36303e76c7af8d2bc10#commitcomment-24710264